Evaluation/Input/Content/Section2_Methods.md

```diff
@@ -1 +0,0 @@
-
```

Comment:
Content files are not mandatory anomore, so we don't need empty files like this one or the famous empty.md 😄